src/librustdoc/html/format.rs

```diff
@@ -912,64 +995,70 @@ fn fmt_type(
 }
 
 impl clean::Type {
-    crate fn print<'b, 'a: 'b, 'tcx: 'a>(
+    crate fn print<'b, 'a: 'b, 'tcx: 'a, 'c: 'a>(
         &'a self,
         cache: &'b Cache,
         tcx: TyCtxt<'tcx>,
+        relative_to: &'c Vec<String>,
```

Comment:
I think you can remove this lifetime parameter too. Actually I don't think you even need to mark it as `&'a`.
```suggestion
    crate fn print<'b, 'a: 'b, 'tcx: 'a>(
        &'a self,
        cache: &'b Cache,
        tcx: TyCtxt<'tcx>,
        relative_to: &Vec<String>,
```